notebooks/building_production_ml_systems/labs/1_training_at_scale_vertex.ipynb

```diff
@@ -8,7 +8,8 @@
     "**Learning Objectives:**\n",
```

Comment:
<p>To be very clear, instead of using bullet points can you use a, b to correspond to the Todos in the cell below?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/647/discussion/#comment-2323486225'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>